en/editor/project/index.md

```diff
@@ -68,6 +68,94 @@ The name of the compressed texture is only used for display. When adding a compr
 
 If you want to replace all the preset options currently used for image asset, you can move the mouse to the preset name, click the button to copy the ID to and search and replace in the project by yourself.
 
+### Export / import compressed texture presets
+
+The compressed texture configuration page allows you to import and export compressed texture presets for better cross-project reuse configuration, or you can edit the compressed texture presets externally and import them into the editor.
+
+In most cases, you can import and export directly. If you need to write this configuration yourself, you need to refer to the following interface definitions and examples:
```

Comment:
In most cases, you can import and export directly. If you need to write this configuration yourself, you need to refer to the following interface definitions and examples:
-->
In most cases, importing and exporting directly works well. If, however, the configuration needs to be customized, please refer to the following interface definitions and examples: